bin/cfndsl

```diff
@@ -95,13 +99,18 @@ if options[:update_spec]
   STDERR.puts "Specification successfully written to #{CfnDsl.specification_file}"
 end
 
+if options[:assetversion]
+  spec_file = JSON.parse File.read(CfnDsl.specification_file)
+  STDERR.puts spec_file['ResourceSpecificationVersion']
+end
```

Comment:
I was thinking here. But I see you included similar logic on line 113 instead. That works even better.